### DISCUSSION TITLE
Difference between 'bank' as used by the Fed and FIBO's Bank class

### DISCUSSION THREAD

**Discussion by rivettp** [edmcouncil/fibo#1391] [Financial Business and Commerce (FBC)] [answered]
The term 'bank' as used by the Fed and FIBO FRSMember Bank, includes some other institutions like investment banks, which are not the same as FDIC insured banks, so that's why it is a subclass of financial institution rather than bank.

We should at least have an explanatory note but ideally separate out GovernmentInsuredBank as a separate class.

_Originally posted by @ElisaKendall in https://github.com/edmcouncil/fibo/diffs_

**Comment by ElisaKendall** [comment, marked as answer]
FRS members are not all banks, and certainly not all government insured banks - some of them are insurance companies, for example.

The only possible change I would consider here is to change the name from FRSMemberBank to FRSMember, and from FRSNonMemberBank to FRSNonMember, although that's not the terminology the Fed uses.  But technically it would be correct.

**Comment by rivettp** [comment]
That would make sense, we could add the FRS term as a synonym.
Also should we use equivalentClass restrictions to automatically classify as FRSMember based on hasRegulator FRS?
In fact do we even need FRSNonMember(Bank) which it seems is the complement? Would all Institutions all over the world not be instances of this? If that's not the case then the definition needs clarifying.
